### DISCUSSION TITLE
My AllSky camera setup

### DISCUSSION THREAD

**Discussion by ericofpendom** [aaronwmorris/indi-allsky#1516] [Show and tell]
I really hope you don't mind me showing my camera setup here. I am really new to this and I have only had the camera in place for 3 days now. I am currently using the Thomas Jacquin AllSky software and I am having a couple of issues so I have been searching today for any alternatives. I have come across INDI-AllSky and it looks great.
Anyway this is my setup. I know it is quite unconventional but I thought it might work.
I have a Raspberry Pi4 4GB running it with a 120 degree wide, 12MP lens, and I have two temp/humidity sensors attached to an Arduino Nano, connected to the Pi's USB. This monitors the temps and controls the heater ring and fan. The Pi and the Arduino are in the loft space along with the power supply, (an old PC PSU) all out of the rain.
As I say, it is very new and I am looking forward to getting some great images.
Thanks for looking and I welcome any feedback, positive or not.

![IMG_1653](https://github.com/user-attachments/assets/352046fc-ac4d-4f3b-9c67-b4d1c8041c48)
![IMG_1654](https://github.com/user-attachments/assets/7b1b4c09-23f0-4b9c-a3aa-698e017df072)
![IMG_1655](https://github.com/user-attachments/assets/54c3444d-812f-488a-beaa-eed70ef513cf)
![IMG_2096](https://github.com/user-attachments/assets/d94bd530-e54f-4858-b44a-d2e516e51840)

**Comment by bluthen** [comment]
Some neat ideas there. I'm glad you shared. You have not noticed any problems with the long ribbon cable? I tried using one for a polar alignment camera once and I think the long ribbon cable was an issue for me.

**Reply by ericofpendom** [reply]
I have used one on the Pi for a couple of years now, nothing serious, but never had any issues. It is 2mtrs long. Someone has suggested that the direct sunlight might be an issue but we don't get enough of that here in the NW UK.

**Comment by quanticleap** [comment]
Hi, which schema are you using for controlling fan and dew heater?arduino uno use mqtt to comunicate with raspberry pi?Thanks
